server/sonar-db-dao/src/main/java/org/sonar/db/DbClient.java

```diff
@@ -367,6 +370,8 @@ public DefaultQProfileDao defaultQProfileDao() {
     return defaultQProfileDao;
   }
 
+  public EsQueueDao esQueueDao() { return esQueueDao; }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)